UiKitDz/UiKitDz/FeedViewController.swift

```diff
@@ -0,0 +1,123 @@
+// FeedViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Oсновной экран с лентой рекомедациями и сторисами
+final class FeedViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let leftLabelText = "RMLink"
+        static let fontDancingScript = "DancingScript-Medium"
+        static let sizeTextLeftLabel = 22
+        static let storiesCellIdentifier = "StoriesTableViewCell"
+        static let mainCellIdentifier = "MainTableViewCell"
+        static let recomendCellIdentifier = "RecomendTableViewCell"
+        static let subscribeButtonCornerRadius = 8
+        static let cancelButtonImage = "xmark"
+    }
+
+    private let tableView: UITableView = .init()
+    private let userStories = SourseTable.makeUsers()
+    private let userPosts = SoursePosts.makeImage()
+    private let recomendedUsers = SourseRecomend.makeFriends()
+
+    private let sections: [CellTypePost] = [.stories, .firstPage, .recomed, .pages]
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureView()
+        configureStoryCell()
+    }
+
+    // MARK: - Private Methods
+
+    private func configureView() {
```

Comment:
```suggestion
    private func configureView() {
```
  тут больше похоже на кофигур навигейшн бара